package.json

```diff
@@ -28,7 +31,7 @@
     "@types/gunzip-maybe": "^1.4.0",
     "@types/rimraf": "^3.0.2",
     "@types/tar-stream": "^2.2.2",
-    "gunzip-maybe": "^1.4.2",
+    "prettier": "^3.2.5",
```

Comment:
I've just run prettier 2.8.8 against main and merged the diffs.

So on your branch right now, `pnpm test:format` should FAIL because it's detecting formatting changes.

After you merge `main` again, it should pass, and `pnpm prettier` should generate no diffs